src/Routing/Middleware/AssetMiddleware.php

```diff
@@ -39,26 +38,6 @@ class AssetMiddleware
      */
     protected $cacheTime = '+1 day';
 
-    /**
-     * A extension to content type mapping for plain text types.
-     *
-     * Because finfo doesn't give useful information for plain text types,
-     * we have to handle that here.
-     *
-     * @var array
-     */
-    protected $typeMap = [
-        'css' => 'text/css',
-        'json' => 'application/json',
-        'js' => 'application/javascript',
-        'ico' => 'image/x-icon',
-        'eot' => 'application/vnd.ms-fontobject',
-        'svg' => 'image/svg+xml',
-        'html' => 'text/html',
-        'rss' => 'application/rss+xml',
-        'xml' => 'application/xml',
-    ];
```

Comment:
Does symfony handle this now?

Reply:
No, this not directly related to using the symfony classes. 

Maintaining as map of extension to mime types in the middleware was redundant since we already have such a map in our `Response` class and can use `Response::getMimeType()` to get mimetype for an extension.